utils/mixin/_text-ellipsis.scss

```diff
@@ -0,0 +1,17 @@
+/**
+ * Text Ellipsis
+ *
+ * @param {number} $lines - Numbers of lines to make the text overflow ellipsis
+ */
+@mixin text-ellipsis($lines: 1) {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	@if $lines > 1 {
+		display: -webkit-box;
+		-webkit-line-clamp: $lines;
+		-webkit-box-orient: vertical;
+		visibility: visible;
```

Comment:
Why do we also set the visibility?

Reply:
Would recommend adding `overflow: hidden;`

Reply:
The overflow hidden is above in the default parameters, and the visibility visible is an ios issue that I solved with the visibility, I think it's related to gsap and all the inline styles that gsap adds.